### PR TITLE
Reduce steps count to list the env vars

### PR DESCRIPTION
**Pull request by mbelsky**
Hey,

It's just a change that helps list env vars two steps faster

### CODE REVIEW

File: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 1]
```diff
@@ -47,17 +47,10 @@ Pod:
     envar-demo      1/1       Running   0          9s
     ```
 
-1. Get a shell to the container running in your Pod:
+1. List the pod's container environment variables:
```
Comment (sftim) [inline content/en/docs/tasks/inject-data-application/define-environment-variable-container.md:50]:
```suggestion
1. List the Pod's container environment variables:
```

**Review by sftim — COMMENTED** [review]
Makes sense, nicely concise.
/lgtm

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-24685--kubernetes-io-master-staging.netlify.app/docs/tasks/inject-data-application/define-environment-variable-container/

**Review by irvifa — COMMENTED** [review]
/approve

**Pull request merged**